### DISCUSSION TITLE
Feature request: Unable to connect to other nodes

### DISCUSSION THREAD

**Discussion by cyndane31** [dougiteixeira/proxmoxve#123] [Feature request] [answered]
I have multiple nodes running in a cluster. I'm able to add my primary node no problem, but when I attempt to add the others, I receive an "unable to connect" error. Do I need to create another user account for each node I want to monitor?

**Comment by dougiteixeira** [comment]
I'm working on the integration to allow adding multiple nodes to the same configuration entry, soon I'll make it available for you to test.

**Comment by dougiteixeira** [comment, marked as answer]
Ei @cyndane31 , I made some changes that allow selecting multiple nodes and the VM/container to be automatically updated on the running node.

But I don't have clusters configured, can you test the beta version [2.0.0.b4](https://github.com/dougiteixeira/proxmoxve/releases/tag/2.0.0.b4) and give feedback on how it works?
